cli/template/extras/config/prettier.config.ts

```diff
@@ -0,0 +1,5 @@
+import { type Config } from "prettier";
+
+export default {
+  plugins: [require.resolve("prettier-plugin-tailwindcss")],
```

Comment:
Why do we have this require.resolve here? Just the string works

Reply:
whops you got to that before me https://github.com/t3-oss/create-t3-app/pull/1500#discussion_r1254026966